plugins/tiddlywiki/multiwikiclient/config-incoming-updates-filter.tid

```diff
@@ -0,0 +1,2 @@
+title: $:/config/multiwikiclient/incoming-updates-filter
+text: [all[]] -[[$:/isEncrypted]] -[prefix[$:/temp/]] -[prefix[$:/status/]] -[[$:/boot/boot.js]] -[[$:/boot/bootprefix.js]] -[[$:/library/sjcl.js]] -[[$:/core]] -[[$:/StoryList]] -[[$:/HistoryList]]
```

Comment:
IMO to should be `-[prefix[$:/StoryList]]` and `-[prefix[$:/HistoryList]]`

Reply:
Thanks @pmario, fixed in d51ad80f8001f774ec98371a542aa10eba3c3dff